Dockerfile

```
@@ -1,16 +1,129 @@
-FROM ubuntu:18.04
-MAINTAINER datapunt@amsterdam.nl
+FROM ubuntu:22.04
+LABEL maintainer="datapunt@amsterdam.nl"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV TZ Europe/Amsterdam
+
+RUN apt-get -y update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        gettext \
+        xz-utils \
+        cmake \
+        gcc \   
+        g++ \
+        libfreetype6-dev \
+        libglib2.0-dev \
+        libcairo2-dev \
+        git \
+        locales \
+        make \
+        patch \
+        openssh-server \
+        protobuf-compiler \
+        protobuf-c-compiler \
+        software-properties-common \
+        wget && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN update-locale LANG=C.UTF-8
+
+ENV HARFBUZZ_VERSION 2.8.2
+
+RUN cd /tmp && \
+        wget https://github.com/harfbuzz/harfbuzz/releases/download/$HARFBUZZ_VERSION/harfbuzz-$HARFBUZZ_VERSION.tar.xz && \
+        tar xJf harfbuzz-$HARFBUZZ_VERSION.tar.xz && \
+        cd harfbuzz-$HARFBUZZ_VERSION && \
+        ./configure && \
+        make && \
+        make install && \
+        ldconfig
+
+RUN apt-get -y update && \
+    apt-get install -y --no-install-recommends \
+        libcurl4-gnutls-dev \
+        libfribidi-dev \
+        libgif-dev \
+        libjpeg-dev \
+        libpq-dev \
+        librsvg2-dev \
+        libpng-dev \
+        libfreetype6-dev \
+        libjpeg-dev \
+        libexempi-dev \
+        libfcgi-dev \
+        libgdal-dev \
+        libgeos-dev \
+        libproj-dev \
+        librsvg2-dev \
+        libprotobuf-dev \
+        libprotobuf-c-dev \
+        libprotobuf-c1 \
+        libxslt1-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get -y update --fix-missing
+
+RUN git clone --single-branch -b pdok-7-6-4-patch-5 https://github.com/pdok/mapserver/ /usr/local/src/mapserver
```

```suggestion
RUN git clone --depth=1 --single-branch -b pdok-7-6-4-patch-5 https://github.com/pdok/mapserver/ /usr/local/src/mapserver
```

Takes about a minute less to download (MapServer has a LONG history).